src/Acuminator/Acuminator.Utilities/Roslyn/Syntax/RoslynSyntaxUtils.cs

```diff
@@ -12,7 +14,7 @@ namespace Acuminator.Utilities.Roslyn.Syntax
 {
 	public static class RoslynSyntaxUtils
 	{
-		public static bool IsLocalVariable(this SemanticModel semanticModel, MethodDeclarationSyntax containingMethod, string variableName)
+		public static bool IsLocalVariable(this SemanticModel semanticModel, MethodDeclarationSyntax containingMethod, string? variableName)
 		{
 			semanticModel.ThrowOnNull(nameof(semanticModel));
```

Comment:
I left null checks here and in other places for these reasons:

1. Nullability analysis is added gradually to Acuminator during the work on other items. It is added only to some files.
 The old code still calls APIs without a nullability analysis, it has not adopted nullable reference types yet. 
 So we need to check args passed to the API since they may come from the old code.

2. For utility methods and libraries it is possible that they code will be called from `#nullable disable` context. So it's safer to check in my opinion.

2.  Nullability analysis at best will give you compiler warnings that in a standard build configuration won't break the compilation AFAIK. I don't really like to rely only on warnings that could be ignored.